tuf/keys.py

```diff
@@ -94,8 +96,9 @@
 except ImportError: # pragma: no cover
   pass
 
-# Import the pyca/Cryptography library so that RSA and generate-purpose
-# cryptography primitives are supported. 
+# Try to import TUF's pyca/Cryptography module (pyca_crypto_keys.py), and
+# verify and record that pyca/Cryptography's library is available.
+# pyca_crypto_keys.py is used for general-purpose cryptography and RSA.
 try:
   import cryptography
```

Comment:
This import is unnecessary and adds nothing. Ditto line 93 above (import Crypto). [Albert's comment](https://github.com/theupdateframework/tuf/pull/289/files#r48359877) on PR #289 likely was trying to make this point.
 
0. This file per se does not use 'cryptography'.
1. Importing tuf.pyca_crypto_keys should (and does, I think - will test) import whatever it, in turn, needs.
2. If importing tuf.pyca_crypto_keys fails, then we already know that we cannot use it and thereby will not be supporting 'cryptography'.
3. This also doesn't help the user figure out what's missing because they're not seeing any messages if either import fails (which is fine).

So nothing is added.